application/admin/views.py

```diff
@@ -34,7 +34,7 @@ def users():
 def user_by_id(user_id):
     user = User.query.filter_by(id=user_id).one()
     if user.user_type == TypeOfUser.DEPT_USER:
-        latest_measure_versions = page_service.get_latest_version_of_all_measures(include_drafts=True)
+        latest_measure_versions = page_service.get_latest_version_of_all_measures(include_not_published=True)
```

Comment:
Given how important this boolean is, I think I’d be tempted to make this more explicit by having two separate methods, `get_latest_published_version_of_all_measures` and `get_latest_version_of_all_measures`?

Reply:
Hmm, fair point. I _think_ it's ok, although I've inverted the default value of the bool as I think the name of the method implies that it should get them all by default. Hopefully we can trust the future developer to read the method signature at least. :)